diff --git a/dist/index.js b/dist/index.js
index b83ca84..ff43fcc 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -15,6 +15,10 @@ class PortableMaker extends maker_base_1.default {
     isSupportedOnCurrentPlatform() {
         return true;
     }
+    clone() {
+        const MakerClass = this.constructor;
+        return new MakerClass(this.configOrConfigFetcher, this.platformsToMakeOn);
+    }
     async make(options) {
         var _a;
         if (options.targetPlatform !== "win32") {
